docs/en/operations/settings/settings.md

```diff
@@ -4805,6 +4805,183 @@ a	Tuple(
 
 If a table has a space-filling curve in its index, e.g. `ORDER BY mortonEncode(x, y)`, and the query has conditions on its arguments, e.g. `x >= 10 AND x <= 20 AND y >= 20 AND y <= 30`, use the space-filling curve for index analysis.
 
+## query_plan_enable_optimizations {#query_plan_enable_optimizations}
```

Comment:
Not sure about documenting the settings. They are too low level for users to deal with. So, having them is good for debugging and disabling some broken optimizations, but for this having them in settings is just enough.

cc @KochetovNicolai

Reply:
I generally like to read the markdown/html docs about settings. It is hard to draw a hard line between public and dev settings, unfortunately.

In particular setting `query_plan_enable_optimizations` (which existed before) was undocumented. One could argue that it is general-purpose enough to have it documented.

But I am so fine with not documenting the individual settings.

Reply:
I am ok do document those settings. Maybe with some disclaimer that they are for debugging only, supposed to be used mainly by developers, do not imply any backward compatibility and might be changed/removed in any time.

Reply:
Done, tnx.